### PR TITLE
add example of hostpath and nfs pv/pvc usage

### PR DESCRIPTION
**Pull request by volume-ji**
I Verified the correctness of the filesystem mount, for example hostPath and nfs, adding two example file.

---
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/788)

### CODE REVIEW

File: examples/ubuntu-vm-hostpath-pv.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  # Set volumeMode to Filesystem so the VM can consume this PV
+  # as a Filesystem device
+  volumeMode: Filesystem
+  hostPath:
+    # Use a path that's visible inside the Virtlet pod.
+    # 9p mode must be supported in host kernel(command: modinfo 9p)
```
Comment (jellonek) [inline examples/ubuntu-vm-hostpath-pv.yaml]: Host kernel? Or should that be supported by guest kernel in VM image?

File: examples/ubuntu-vm-nfs-pv.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  # Set volumeMode to Filesystem so the VM can consume this PV
+  # as a Filesystem device
+  volumeMode: Filesystem
+  nfs:
+    # Use a path exposed in nfs server which has a ip 172.21.0.2(command: showmount -e 172.21.0.2)
+    # 9p mode must be supported in host kernel(command: modinfo 9p)
```
Comment (jellonek) [inline examples/ubuntu-vm-nfs-pv.yaml]: Same as above.

File: examples/ubuntu-vm-hostpath-pv.yaml (hunk 1 above)
Comment (volume-ji) [reply examples/ubuntu-vm-hostpath-pv.yaml]: great, you are right, guest kernel also must support 9p, the example use ubuntu os which support 9p. actually，I want to say that host kernel must install 9p mode. 
I shoud modify the annotation as "# 9p mode must be installed in host kernel(command: modinfo 9p)", is ok?

Comment (jellonek) [reply examples/ubuntu-vm-hostpath-pv.yaml]: Is it really needed on host? It's served by qemu which afaik do not uses any part of 9p from host kernel, it implements itself 9p protocol.

Comment (volume-ji) [reply examples/ubuntu-vm-hostpath-pv.yaml]: OK, you are great again, I used "rmmod 9p" and test again, it works. I change the annotation "# 9p mode must be supported in guest kernel in VM image."

File: examples/ubuntu-vm-nfs-pv.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: nfs-dir-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  # Set Filesystem to Filesystem so the VM can consume this PVC
```
Comment (jellonek) [inline examples/ubuntu-vm-nfs-pv.yaml]:
```suggestion
  # Set volumeMode to Filesystem so the VM can consume this PVC
```

Comment (volume-ji) [reply examples/ubuntu-vm-nfs-pv.yaml]: sorry, it is my fault

Comment (jellonek) [reply examples/ubuntu-vm-nfs-pv.yaml]: Don't be offended, it's not a problem :)

**Pull request merged**